fix: add docs for starting ephemeral/persistent workspaces (#2448)
codenvy/che-docs: modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc

:_content-type: CONCEPT
:description: URL parameter starting ephemeral workspaces
:keywords: ephemeral-workspace, persistent-workspace
:navtitle: URL parameter starting ephemeral workspaces
:page-aliases:

[id="url-parameter-for-starting-ephemeral-workspaces_{context}"]
= URL parameter for starting ephemeral workspaces

> If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} will not persist by default the workspace and its data and they'll be lost when the workspace stops.

Reviewer: ```suggestion
If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} does not persist by default. The data in the workspace are lost when the workspace stops.
```

Author: @deerskindoll thanks for the review,

Here I meant that the workspace itself won't be lost, but only its data. Could you update the suggestion in this way?

Reviewer: sure!

Reviewer 2: @akurinnoy weird, this sentence looks ambiguous. AFAIK if the URL parameter is not part of the URL the default storage type will be used, no?

> The URL parameter for specifying a supported storage Type is `storageType=__<storage_type>__`:

Reviewer: ```suggestion
The URL parameter for specifying a supported storage type is `storageType=__<storage_type>__`:
```
